tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -31,77 +31,6 @@
 from inspirehep.dojson.utils import validate
 
 
-def test_location_from_034__d_f():
-    schema = load_schema('institutions')
-    subschema = schema['properties']['location']
-
-    snippet = (
-        '<datafield tag="034" ind1=" " ind2=" ">'
-        '  <subfield code="d">6.07532</subfield>'
-        '  <subfield code="f">50.7736</subfield>'
-        '</datafield>'
-    )  # record/902624
-
-    expected = {
-        'longitude': 6.07532,
-        'latitude': 50.7736,
-    }
-    result = institutions.do(create_record(snippet))
-
-    assert validate(result['location'], subschema) is None
-    assert expected == result['location']
-
-
-def test_no_location_from_034__d():
-    snippet = (
-        '<datafield tag="034" ind1=" " ind2=" ">'
-        '  <subfield code="d">6.07532</subfield>'
-        '</datafield>'
-    )  # synthetic data
-
-    result = institutions.do(create_record(snippet))
-
-    assert 'location' not in result
-
-
-def test_no_location_from_034__f():
-    snippet = (
-        '<datafield tag="034" ind1=" " ind2=" ">'
-        '  <subfield code="f">50.7736</subfield>'
-        '</datafield>'
-    )  # synthetic data
-
-    result = institutions.do(create_record(snippet))
-
-    assert 'location' not in result
-
-
-def test_no_location_from_invalid_034__d_f():
-    snippet = (
-        '<datafield tag="034" ind1=" " ind2=" ">'
-        '  <subfield code="d">foo</subfield>'
-        '  <subfield code="f">bar</subfield>'
-        '</datafield>'
-    )  # synthetic data
-
-    result = institutions.do(create_record(snippet))
-
-    assert 'location' not in result
-
-
-def test_no_location_from_034__double_d():
-    snippet = (
-        '<datafield tag="034" ind1=" " ind2=" ">'
-        '   <subfield code="d">32.540776</subfield>'
-        '   <subfield code="d">15.561010</subfield>'
-        '</datafield>'
-    )  # record/1442294
-
-    result = institutions.do(create_record(snippet))
-
-    assert 'location' not in result
-
-
```

Comment:
In the end I just wrote a simpler test that doesn't test all of the above because I don't believe that we're going to regress on this.